br/use-br-command-line-tool.md

```diff
@@ -234,6 +236,7 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 br backup raw --pd $PD_ADDR \
     -s "local://$BACKUP_DIR" \
     --start 31 \
+    --ratelimit 120 \
```

Comment:
why these three are 120 while the rest are 128

Reply:
I also want to ask this question, I just fill in this value according to the context

Reply:
@kennytm @3pointer PTAL